src/release/breaking-changes/tooltip-semantics-order.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Accessibility traversal order of tooltip changed 
+description: >
+  the Tooltip widget's message immedately follows the Tooltip widget's child 
+  during accessibility traversal.
+---
+
+## Summary
+
+During accessibility traversal, `Tooltip.message` is visited immedately after
+`Tooltip.child`.
+
+## Background
+
+The `Tooltip` widget usually wraps an interactive UI component such as a button,
+and shows a help message when long pressed. 
+When the message is visible, assistive technologies should announce it after
+the button.
+
+The `Tooltip` widget originally put`Tooltip.message` on an `OverlayEntry` when 
+long pressed.
+As a result, the generated semantics tree does not put `Tooltip.message` 
+immediately after `Tooltip.child`.
+
+You may see accessibility test failures when a tooltip message is visible.
+This is because this update moved the tooltip message in the semantics tree.
+
+## Migration guide
```

Comment:
Maybe we should mention the old behavior is no longer supported.

Reply:
I'm not sure we should say the current behavior is never making a comeback? The new behavior makes more sense as the default but maybe in the future it will be made configurable to allow developers to specify who the accessibility parent is, the overlay or the overlayportal?

Reply:
but for now there is no way to get back to original behavior, and I think we should communicate it more clearly. If we were to update the behavior later, we can update this doc with a new revision.

Reply:
I found it pretty hard to describe what "the old behavior" is so I added "adopt the new semantics order" in your tests.